sql/updates/latest-dev.sql

```diff
@@ -0,0 +1,92 @@
+-- add a new column creation_time to chunk catalog table
+ALTER TABLE _timescaledb_catalog.chunk ADD COLUMN creation_time TIMESTAMPTZ;
+UPDATE _timescaledb_catalog.chunk SET creation_time = now();
```

Comment:
I guess we can do a better assumption here using for example `pg_stat_file` function, something like:

```suggestion
UPDATE
    _timescaledb_catalog.chunk c
SET
    creation_time = (pg_catalog.pg_stat_file(pg_catalog.pg_relation_filepath(r.oid))).change
FROM
    pg_class r, pg_namespace n
WHERE
    r.relnamespace = n.oid
    AND r.relname = c.table_name
    AND n.nspname = c.schema_name
    AND r.relkind = 'r'
    AND c.dropped IS FALSE;
UPDATE _timescaledb_catalog.chunk SET creation_time = now() WHERE creation_time IS NULL;
```
The second UPDATE is just to make sure we'll don't left any record without creation time information. One improvement is try to use pg_stat_file.creation field when the update run on Windows platform. https://www.postgresql.org/docs/15/functions-admin.html